frontend/src/stores/levels/actions.ts

```diff
@@ -13,4 +13,26 @@ async function getLevelsByRoleId(roleId: number): Promise<Level[] | undefined> {
   return role.levels
 }
 
-export { getLevelsByRoleId }
+async function getScoreResponse(
+  roleId: number
+): Promise<{ currentLevel: Level; nextLevel: Level } | undefined> {
```

Comment:
Here you should create also a new interface under domain folder, called QuizResult (containing the currentLevel and the nextLevel.
```suggestion
): Promise<QuizResult> {
```